packages/rocketchat-ui-message/client/messageBox.coffee

```diff
@@ -165,7 +166,7 @@ Template.messageBox.events
 				e.preventDefault()
 				files.push
 					file: item.getAsFile()
-					name: 'Clipboard'
+					name: 'Clipboard - ' + moment().format('L, LTS')
```

Comment:
I think its better use "Time Format" setting instead of hardcode the format here.
Use `RocketChat.settings.get('Message_TimeFormat')` to get the setting value.